### PR TITLE
Fix/show invite tab

### PR DESCRIPTION
**Pull request by a-kuharenko**

### CODE REVIEW

File: server/api/services/user.service.js [hunk 1]
```diff
@@ -54,6 +54,20 @@ const updateUserSettings = async ({ id, settings }) => {
 
 const getStars = async username => StarRepository.getStars(username);
 
+const getUsersToInviting = async ({ orgID, username }) => {
+  const allUsers = await UserRepository.findUserByLetter(username);
+  const orgUsers = await OrgUserRepository.getAllOrganizationUsers(orgID);
+  const usersIdInOrg = orgUsers.map(({ userId }) => userId);
+
+  const users = allUsers
+    .filter(({ id }) => {
```
Comment (TheSubliminal) [inline server/api/services/user.service.js]:
```suggestion
    .filter(({ id }) => !usersIdInOrg.includes(id))
```

Comment (a-kuharenko) [reply server/api/services/user.service.js]: Changed, thanks

File: server/data/repositories/user.repository.js [hunk 2]
```diff
@@ -58,6 +61,17 @@ class UserRepository extends BaseRepository {
       }
     });
   }
+
+  findUserByLetter(username) {
+    return this.model.findAll({
+      where: {
+        username: {
+          [Op.startsWith]: username
```
Comment (OlgaGnatenko) [inline server/data/repositories/user.repository.js]: Will this search handle both uppercase and lowercase? Our usernames are case sensitive, Op.startsWith is also case sensitive. It depends on the requirements. As a user, I would expect that search would return both lowercase and uppercase versions.
https://stackoverflow.com/questions/42352090/sequelize-find-all-that-match-contains-case-insensitive Here's a suggestion how to search regardless of case. Not sure if we need to implement it though, but could be useful.

Comment (a-kuharenko) [reply server/data/repositories/user.repository.js]: This is really important, thanks. 
Fixed

**Pull request merged**